scripts/vagrant-setup-env.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+echo 'Setting up correct env. variables'
+echo "export GOPATH=$GOPATH" >> $HOME/.bash_profile
+echo "export PATH=$PATH:$GOPATH/bin:/usr/local/go/bin" >> $HOME/.bash_profile
+
+source ~/.bash_profile
+
+set -euo pipefail
```

Comment:
@schu `-e` and `-o pipefail` can be moved, but `-u` cannot, because we would otherwise meet an error `"PS1: unbound variable"` inside `.bash_profile`. So I let `-u` stay below `"source ~/.bash_profile"`, and added a comment.